farm/contract/lib.rs

```diff
@@ -73,6 +73,12 @@ mod farm {
 
         /// Reward rates per user of unclaimed, accumulated users' rewards.
         pub user_claimable_rewards: Mapping<UserId, Vec<u128>>,
+
+        /// Flag indicating whether farm is active.
+        /// Farm is active when:
+        /// * farm is running
+        /// * farm is planned for the future
+        pub is_active: bool,
```

Comment:
I suspect it would be useful to have a getter message for `is_active` so that the frontend can call it.

Reply:
Added to `FarmDetails`.

Reply:
Done in [Add is_active to FarmDetails](https://github.com/Cardinal-Cryptography/common-amm/pull/86/commits/e508a1619cb15dc60cc335bfb919112022a85fe0).